src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Properties.cs

```diff
@@ -115,19 +113,15 @@ private void ShadowPropertyChanged(object? sender, PropertyChangedEventArgs e)
 			return;
 		}
 
-		if (UI.Shadow.IsShadowSizeProperty(e.PropertyName))
-		{
-			OnShadowSizeChanged();
-		}
-
-		InvalidateFromShadowPropertyChange();
+		OnShadowSizeChanged();
 	}
 
 	private void OnShadowSizeChanged()
 	{
 		if (_currentContent != null && _currentContent.ActualWidth > 0 && _currentContent.ActualHeight > 0)
 		{
-			UpdateCanvasSize(_currentContent.ActualWidth, _currentContent.ActualHeight, Shadows);
+			InvalidateFromShadowPropertyChange();
+			this.GetDispatcherCompat().Schedule(() => UpdateCanvasSize(Shadows));
```

Comment:
Won't this cause two schedules? One from the `UpdateShadows` call, and one here?